migrations/disable-erlang-support.js

```diff
@@ -5,9 +5,9 @@ module.exports = {
   created: new Date(2016, 10, 5),
   run: function(callback) {
     db.request({
-      db: '_config',
+      db: '_node',
       method: 'DELETE',
-      path: 'native_query_servers/erlang',
+      path: 'couchdb@localhost/_config/native_query_servers/erlang',
```

Comment:
I made a comment about it on the actual commit, but I'm just addressing the local node here because they are migrations so it should be fine: it's not long living code that gets run all the time.